### PR TITLE
Packagify unzipfile

### PR DESCRIPTION
**Pull request by jochman**
## Status
Ready

## Description
Just packagify

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Scripts/UnzipFile/UnzipFile.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  id: UnzipFile
+  version: -1
+enabled: true
+name: UnzipFile
+outputs:
+- contextPath: ExtractedFiles
+  description: A list of file names that were extracted from the ZIP file.
+  type: Unknown
+script: '-'
+subtype: python3
```
Comment (Itay4) [inline Scripts/UnzipFile/UnzipFile.yml]:
```suggestion
subtype: python2
```

File: Scripts/UnzipFile/UnzipFile.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+system: false
+tags:
+- Utility
+- file
+timeout: '0'
+type: python
+dockerimage45: demisto/7zip
```
Comment (Itay4) [inline Scripts/UnzipFile/UnzipFile.yml]: why ?

Comment (jochman) [reply Scripts/UnzipFile/UnzipFile.yml]: no tag, so I'm preserving bc.

Comment (Itay4) [reply Scripts/UnzipFile/UnzipFile.yml]: `latest` is redundant in this case, lets keep the docker image as is

**Pull request merged**